llvm/test/CodeGen/SystemZ/builtin-longjmp-backchain-01.ll

```diff
@@ -0,0 +1,34 @@
+;Test -mbackchain longjmp load from jmp_buf.
+; Frame pointer from Slot 1.
+; Jump address from Slot 2.
+; Backchain Value from Slot 3.
+; Stack Pointer from Slot 4.
+; Literal Pool Pointer from Slot 5.
+
+; RUN: llc < %s -mtriple=s390x-linux-gnu -O2 | FileCheck %s
+
+@buf = dso_local global [20 x ptr] zeroinitializer, align 8
+
+; Function Attrs: noreturn nounwind
+define dso_local void @foo() local_unnamed_addr #0 {
```

Comment:
See above for removing unnecessary annotations.  Here, we do need the `backchain` attribute, but not the others.  That attribute can be specified inline rather than as part of an attribute bundle - something alone these lines should work:
```
define void @foo() "backchain" {
```